README.md

```diff
@@ -41,6 +41,9 @@ When the app is deployed to a cloud environment, the `DATABASE_URL` `.env` var w
 During the installation process (if you follow the steps on [postgresql.org](https://www.postgresql.org/download/)), you will be prompted to set a password - make sure to use something you'll remember.
 
 ## Starting the App
+
+If NODE is not already installed, to use the Hangar you first need to download it from [here](https://nodejs.org/en/) Be sure to use the version v12.18.3 LTS. Hangar now only supports v12.x. 
```

Comment:
Can you move all of this to a new section right below `# Local Development`? The section should be `## Node` and it should contain the revised text below:
```suggestion
Hangar uses Node.js as its runtime. To use the Hangar, you'll need to [download](https://nodejs.org/en/) and install the LTS (Long Term Support) version, which is currently 12.x. If you decide to use another version of Node such as 13.x or 14.x, you may experience unexpected issues.
```

For the future, avoid using specific versions in documentation (e.g., "v12.18.3") because you never know what could happen to the project. For example, it's not unheard of for projects to outright pull versions because of critical vulnerabilities (i.e., deprecating and pulling the binary for v12.18.3 so it's virtually impossible to install that version).

Reply:
@oscargoddard make sure to review the other comment above about the new section

Reply:
I moved them under ## Node